helmfile/charts/calico-accountant/templates/rbac.yaml

```diff
@@ -1,14 +1,13 @@
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: calico-accountant
+# ---
+# apiVersion: v1
+# kind: ServiceAccount
+# metadata:
+#   name: calico-accountant
```

Comment:
If it's not going to use this SA, and instead use the calico-node then why do we have the rest of this rbac? (Also any reason why it doesn't use it's own SA?)

Reply:
Yea, if it's not used then I'd say that. We can always dig it back again if we want to get it back.

Reply:
Removed the file now and created an issue for fixing these permissions later.
https://github.com/elastisys/compliantkubernetes-apps/issues/1624